drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/referral_posting.rb

```diff
@@ -75,6 +92,15 @@ class ReferralPosting < ::HmisExternalApis::HmisExternalApisBase
     OUTGOING_STATUSES = [:assigned_status, :accepted_pending_status, :denied_pending_status].freeze
     scope :outgoing, -> { where(status: OUTGOING_STATUSES) }
 
+    private def validate_status_change
+      return unless status.present? && status_was.present?
```

Comment:
It looks like status_was has the value of status initially, so the validation would occur even if the status hasn't been changed yet. Maybe add check for that? 
```suggestion
      return unless status_changed? && status.present? && status_was.present?
```